### DISCUSSION TITLE
make a .exe file of this please

### DISCUSSION THREAD

**Discussion by calcahura** [MaxRobinsonTheGreat/EvolutionSimulatorV2#49] [Ideas and Suggestions]
i want an non-internet exe file or version of life engine because my internet on my pc is broken, im using my phone to make this discussion and look, its a long story of how my internet on my pc broke, so please make an non-internet exe file of the life engine.

**Comment by xkortex** [comment]
You could bundle this as an Electron app, and submit a PR ;)

**Reply by syspic** [reply]
if their internet is that broken they arent going to submitting a PR anytime soon lol

**Reply by calcahura** [reply]
> if their internet is that broken they arent going to submitting a PR anytime soon lol

yez i dont know if my parents are going to buy me a new computer or if they will just repair the non-internet computer

**Reply by xkortex** [reply]
> if their internet is that broken they arent going to submitting a PR anytime soon lol

Yes, I'm aware, my comment was thoroughly tongue-in-cheek :p 

It's not entirely the best etiquette in on a FOSS discussion to be like, "hey make this feature for me! pretty please?" It's not super productive. 

Suboptimal: 

> FEATURE: make a .exe file of this please - my internet doesn't work

Better: 

> FEATURE: Build a portable executable for distribution 
> Application is currently only distributed as a webapp. This limits its reach to users who may have limited internet connectivity. A portable distribution, such as Electron app, would increase the reach. *proceed to list pros and cons*

**Comment by xkortex** [comment]
@MaxRobinsonTheGreat  Have you ever used https://github.com/electron/electron-packager ?

Neither have I, but it seems pretty simple to use. I think one would have to restructure slightly cause I get this error popup when I try to start but it seems pretty tractable. 

![image](https://user-images.githubusercontent.com/4248547/144667834-66b002f1-15b1-4bae-b795-f93a1877a2df.png)

**Comment by MaxRobinsonTheGreat** [comment]
I am not sure how you will download an exe if your internet is not working. I see this as very low priority, but @xkortex if you get an electron app working that'd great.
